acquamanCommon.pri

```diff
@@ -153,7 +153,15 @@ HEADERS += \
 	source/ui/dataman/AM2DScanView.h \
 	source/ui/AMDatamanStartupSplashScreen.h \
 	source/dataman/database/AMDbUpgrade.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt1.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt2.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt3.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt4.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt5.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt6.h \
+	source/dataman/CLS/CLSDbUpgrade1Pt1.h \
```

Comment:
@helfrij first of all, I jsut moved the CLSDbUpgrade1Pt1 to clsCommon.pri.
AMDbUpgrades are primarily used for AM related db upgrade
CLSDbUpgrades are for cls specific upgrades
we also have beamline specified upgrades, which live in the dataman/xxx (beamline0